qgepplugin/gui/qgepdatamodeldialog.py

```diff
@@ -0,0 +1,773 @@
+# -*- coding: utf-8 -*-
+# -----------------------------------------------------------
+#
+# Profile
+# Copyright (C) 2012  Matthias Kuhn
```

Comment:
Sure about that?

Reply:
Ok will change (I never understand what these headers a good for, also in that case the origin is probably indeed your file from a copy/paste ?)